boards/arduino_mkrwifi1010/Cargo.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+name = "arduino_mkrwifi1010"
+version = "0.1.0"
+authors = ["Osvald Ivarsson <osvald@osvald.org>"]
+description = "Board Support crate for the Arduino MKR WiFi 1010"
+keywords = ["no-std", "arm", "cortex-m", "embedded-hal", "arduino"]
+categories = ["embedded", "hardware-support", "no-std"]
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/atsamd-rs/atsamd"
+readme = "README.md"
+edition = "2021"
+
+[dependencies.cortex-m-rt]
+version = "0.7"
+optional = true
+
+[dependencies.atsamd-hal]
+version = "0.15"
+default-features = false
+
+[dependencies.usb-device]
+version = "0.2"
+optional = true
+
+[dev-dependencies]
+cortex-m = "0.7"
+panic-halt = "0.2"
+panic-semihosting = "0.5"
+usbd-serial = "0.1"
+wifi-nina = { git = "https://github.com/Foosvald/wifi-nina", branch = "invert-reset-logic-with-feature", features = ["reset-high"] }
```

Comment:
Have you managed to get your changes upstreamed? I don't really like depending on someone's personal branch. That feels very hacky. Plus, I don't think you can even publish to crates.io if you have a Git dependency.